README.md

```diff
@@ -1,7 +1,334 @@
+string organizationId = "Your organization ID";
+ICoveoPlatformConfig config = new CoveoPlatformConfig(Constants.Endpoint.UsEast1.HIPAA_PUSH_API_URL,
+    Constants.PlatformEndpoint.UsEast1.HIPAA_PLATFORM_API_URL,
+    apiKey,
+    organizationId);
+ICoveoPlatformClient client = new CoveoPlatformClient(config);
+```
+
+## Using the SDK to push documents to a Push API source
+Methods to interact with the Push API are part of the `client.DocumentManager` object.
+### Prerequisites
+1. Ensure your API key has access to push documents inside a Push API source.
+  * You can create an API key when creating a Push API source.
+  * You can also create an API key manually. For more information about which privileges are required, visit [Privilege Reference](For more information about which privileges are required, visit [Privilege Reference](https://docs.coveo.com/en/1707/cloud-v2-administrators/privilege-reference#sources-domain).
+1. Create a Push API source. For more information, visit [Add or Edit a Push Source](https://docs.coveo.com/en/1546/cloud-v2-administrators/add-or-edit-a-push-source).
+### Pushing a batch of documents
+For overall performance, it is better to push your documents in batches. Use the single document method when batches are not required to meet both your performance and volume requirements. For more information, visit [Managing Batches of Items in a Push Source](https://docs.coveo.com/en/90/cloud-v2-developers/managing-batches-of-items-in-a-push-source).
+
+```csharp
+PushDocument firstDocumentToAdd = new PushDocument("http://www.coveo.com/page1") {
+    ClickableUri = "http://www.coveo.com/page1",
+    ModifiedDate = DateTime.UtcNow
+};
+
+PushDocument secondDocumentToAdd = new PushDocument("http://www.coveo.com/page2") {
+    ClickableUri = "http://www.coveo.com/page2",
+    ModifiedDate = DateTime.UtcNow
+};
+
+IList<PushDocument> documentsToAdd = new List<PushDocument> {
+    firstDocumentToAdd,
+    secondDocumentToAdd
+};
+
+client.DocumentManager.AddOrUpdateDocuments(sourceId, documentsToAdd, null);
+```
+### Pushing a single document
+You should only use this method when you want to add or update a single document. Pushing several documents using this method may lead to the `429 - Too Many Requests` response from the Coveo platform and decrease the overall performance time of your application.
+
+```csharp
+string sourceId = "your Push API source ID";
+PushDocument document = new PushDocument("https://coveo.com") {
+    ClickableUri = "https://www.coveo.com",
+    ModifiedDate = DateTime.UtcNow
+};
+
+document.AddMetadata("title", "Coveo's home page.");
+
+PushDocumentHelper.SetContent(document, "this is Coveo's website home page.");
+
+client.DocumentManager.AddOrUpdateDocument(sourceId, document, null);
+```
+
+**Good to know:**
+* The `SetContent` method has an overload taking a Stream instead of a string.
+* The PushDocumentHelper class has also the method `SetContentFromFile` taking a file path as an argument.
```

Comment:
"has also the method `SetContentFromFile`..." -> "also has a `SetContentFromFile` method..."

Reply:
```suggestion
* The PushDocumentHelper class also has a  `SetContentFromFile` method taking a file path as an argument.
```